### PR TITLE
Block RPC

### PR DESCRIPTION
**Pull request by tzdybal**
Work in progress.

### CODE REVIEW

**Comment by liamsi** [comment]
rebasing on main should fix your problems with github actions / build on CI

**Review by adlerjohn — COMMENTED** [review]
Looks okay, but waiting for CI to be fixed.

**Comment by liamsi** [comment]
Fixed by your local approvooor for you :-D

File: block/manager.go [hunk 1]
```diff
@@ -109,13 +110,21 @@ func NewManager(
...
+}
+
 func (m *Manager) SetDALC(dalc da.DataAvailabilityLayerClient) {
 	m.dalc = dalc
 	m.retriever = dalc.(da.BlockRetriever)
 }
 
 func (m *Manager) AggregationLoop(ctx context.Context) {
-	timer := time.NewTimer(m.conf.BlockTime)
+	timer := time.NewTimer(0)
```
Comment (mattdf) [inline block/manager.go:127]: What's the reason for removing this?

**Review by mattdf — APPROVED** [review]

**Pull request merged**